static/js/components/ExpandedLearningResourceDisplay.js

```diff
@@ -109,7 +108,7 @@ const ExpandedLearningResourceDisplay = (props: Props) => {
   const url = selectedRun && selectedRun.url ? selectedRun.url : object.url
   const cost = selectedRun ? minPrice(selectedRun.prices, true) : null
 
-  const offeredBy = getPreferredOfferedBy(object.offered_by)
+  const offeredBy = object.offered_by.join(", ")
```

Comment:
Maybe something to check with @pdpinch or @abdulkdawson about - the button to take a course on the drawer display now reads "Take course on MITx, Micromasters" for example if it has two offered_by values, but the link goes to MITx.  Is this the desired behavior?  

<img width="407" alt="Screen Shot 2019-12-30 at 11 51 37 AM" src="https://user-images.githubusercontent.com/187676/71591747-85501f80-2afb-11ea-899c-46133b789da2.png">